localizer:
    files:
        path_loss_file: "pathLoss.csv"
        GP_file: "GP"
    topics:
        output:
            particlecloud_topic: "localizer/particles"
            pose_topic: "localizer/pose"
    frames:
        # Refer to http://wiki.ros.org/amcl
        map_frame: "map"
        odometry_frame: "odom"
        base_frame: "base_link"
        # wifi base frame
        wifi_base_frame: "wifi_base_frame"

    parameters:
        x_min: -20
        x_max: 20
        y_min: -20
        y_max: 20
        n_particles: 500
        rotational_noise: 0.05
        translational_noise: 0.2
        minimum_rss_value: -100
